### PR TITLE
Feature/lab 67 update database

### PR DESCRIPTION
**Pull request by diegodemorais**
@anttoniol @RenanSchwyz @tutss  @robonauta  @AndrewIjano

### CODE REVIEW

File: backend/router.py [hunk 1]
```diff
@@ -1,8 +1,7 @@
-from museu.viewsets import UserViewset, HistoriaViewset
```
Comment (robonauta) [inline backend/router.py:1]: Didn't HistoriaViewSet changed to AppViewSet?

File: museu/viewsets.py [hunk 2]
```diff
@@ -2,9 +2,6 @@
 from . import models
 from . import serializers
 
-class UserViewset(viewsets.ModelViewSet):
-    queryset = models.User.objects.all()
-    serializer_class = serializers.UserSerializer
 
 class HistoriaViewset(viewsets.ModelViewSet):
```
Comment (robonauta) [inline museu/viewsets.py]: Shouldn't this be changed to AppViewSet?

File: backend/urls.py [hunk 3]
```diff
@@ -22,8 +22,7 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('user/', views.UserView.as_view()),
-    path('historia/', views.HistoriaView.as_view()),
+    path('historia/', views.AppView.as_view()),
```
Comment (robonauta) [inline backend/urls.py:23]: Shouldn't we change the route from 'historia' to another thing?

**Review by robonauta — CHANGES_REQUESTED** [review]
I think there are some minor naming inconsistencies that are easily solvable.

File: backend/urls.py (hunk 3 above)
Comment (diegodemorais) [reply backend/urls.py:23]: We decided together to keep 'historia' here for while.

**Review by robonauta — APPROVED** [review]

**Pull request merged**